app/user_test.go

```diff
@@ -530,10 +530,10 @@ func TestRecordUserTermsOfServiceAction(t *testing.T) {
 	defer th.TearDown()
 
 	user := &model.User{
-		Email: strings.ToLower(model.NewId()) + "success+test@example.com",
-		Nickname: "Luke Skywalker", // trying to bring balance to the "Force", one test user at a time
-		Username: "luke" + model.NewId(),
-		Password: "passwd1",
+		Email:       strings.ToLower(model.NewId()) + "success+test@example.com",
+		Nickname:    "Luke Skywalker", // trying to bring balance to the "Force", one test user at a time
+		Username:    "luke" + model.NewId(),
+		Password:    "passwd1",
```

Comment:
Are these spacing changes because of errors reported by `check-style`?
If not I'd suggest reverting them because there are no other changes needed to this file for this branch.

Reply:
It was either check-style (which runs go fmt), or an explicit call to go fmt. In any case, go fmt automatically formats some things sometimes. Would you prefer to not commit these changes?

Reply:
hmm, since the change were to clean up style errors that were reported, I think we can leave the changes. I'm guessing it was this file and 1 other.